ontopy/utils.py

```diff
@@ -109,7 +109,7 @@ def getiriname(iri):
 
 def asstring(  # pylint: disable=too-many-return-statements,too-many-branches,too-many-statements
     expr,
-    link="{lowerlabel}",
+    link="{label}",
```

Comment:
Shouldn't the link still be "the label of the concept in lower case and with spaces replaced with hyphens"?

Reply:
As default? I guess this could be discussed with Jesper. He introduced this recently and I disagree :) Also, right now it doesn't work. I'll make an issue about it.

Reply:
Using lowerlabel introduced an error. So we need to keep label. I have added that lowerlabel makes ontograph stypes fail in the issue.